### ISSUE TITLE
[N64] Clock is wrong on Doubutsu no Mori

### ISSUE THREAD

**Issue by Mastergatto** [higan-emu/ares#1193]
**Describe the bug**
I'm reporting specifically about the N64 clock.

In Doubutsu no Mori, the white-blue cat normally should tell you the current date and time, but he will tell them wrong on Ares. 

For example, the correct date would be 2023/08/13 00:00, but the game thinks its the year 1953! Even the time is wrong, it's like one day, two hours and eleven minutes behind.

**To Reproduce**
1. Play Doubutsu no Mori.
2. It will tell date and/or time wrong.

**Expected behavior**
It should report correct time and date.

**Screenshots**
![Doubutsu no Mori](https://github.com/ares-emulator/ares/assets/1028191/3e530cee-1bbc-4781-bda6-d5ce31b0376b)

**Additional context**
Ares is, as of writing, on the current commit [8476c88](https://github.com/ares-emulator/ares/commit/8476c880bc0bfe4ef2332317856849ab968de919), built from AUR on ArchLinux.

**Comment by rasky** [comment]
Can you confirm it also happens with no saves (that is, no `save.rtc` file)?

**Comment by rasky** [comment]
Also I think the game has a feature to set the clock. Try setting the correct time and see if it sticks correctly after you close and restart Ares.

**Comment by Mastergatto** [comment]
I can confirm that even with no saves, and after I have set the clock the game still thinks it's the year 1953.

![immagine](https://github.com/ares-emulator/ares/assets/1028191/e6922cba-078b-4f91-a3e8-f3c4d725eab4)


As for the 64DD clock, the thing is that I already set it a while ago, although I can't remember if I have done it correctly. Anyways, tonight ( when I opened this issue) I have already deleted the 64DD save.rtc, and then set time and date again. Now, when I load a N64DD game, I can see that the clock is again behind, by like two minutes.

(P.S.: How do I load only the 64DD firmware, without loading also any disk? I forgot it....)
